test/powershell/Modules/Microsoft.PowerShell.Management/Restart-Computer.Tests.ps1

```diff
@@ -107,3 +107,16 @@ finally
     Disable-Testhook -testhookName $restartTesthookName
     Set-TesthookResult -testhookName $restartTesthookResultName -value 0
 }
+
+Describe 'Non-admin on Unix' {
+    BeforeAll {
+        $skip = $false
+        if ($IsWindows -or [environment]::IsPrivilegedProcess -or ($null -eq (Get-Command shutdown -CommandType Application -ErrorAction Ignore))) {
```

Comment:
will this test be properly portable down level? 
It may be good to note that `[environment]::IsPrivilegedProcess` is a dotnet version 8 feature

Reply:
I don't think we are planning on porting this change to older PS7